src/atoms/affine/add_subtract.jl

```diff
@@ -1,53 +1,30 @@
-#############################################################################
-# add_subtract.jl
-# Handles unary negation, addition and subtraction of variables, constants
-# and expressions.
-# All expressions and atoms are subtpyes of AbstractExpr.
```

Comment:
Is this not useful or has it be moved somewhere ?

Reply:
I didn't think it was useful. It's pretty obvious that `NegateAtom` handles negation. And we really need a docstring for each atom with examples, rather than this vague comment in the file. But that's a separate PR...